crates/runtime/src/dataconnector/github/mod.rs

```diff
@@ -404,9 +410,15 @@ pub fn parse_globs(input: &str) -> super::DataConnectorResult<Arc<GlobSet>> {
     Ok(Arc::new(glob_set))
 }
 
+enum GitHubFilterRemap {
+    Column(&'static str),
+    Operator((Operator, &'static str)),
+}
+
 struct GitHubPushdownSupport {
     ops: Vec<Operator>,
-    remap: Option<&'static str>,
+    remaps: Option<Vec<GitHubFilterRemap>>,
+    uses_modifiers: bool,
```

Comment:
What does `uses_modifiers` mean?

Reply:
Whether the query parameter supports e.g. `<` or `>` in the query string. I've added some comments inline as well.